website/blog/2023-12-15-ferretdb-neon-add-mongodb-compatibility.md

```diff
@@ -0,0 +1,236 @@
+By using FerretDB, developers can [access familiar MongoDB features and tools using the same syntax and commands](https://blog.ferretdb.io/mongodb-crud-operations-with-ferretdb/) for many of their use cases.
+
+<!--truncate-->
+
+[Neon](https://neon.tech/) is an open source, fully managed serverless Postgres which is licensed under the Apache 2.0 license.
+What makes Neon unique is that it separates storage and compute, which makes it possible to scale up compute based on demand.
+It also supports branching, which allows you to create a copy of your database in seconds for testing and development.
+
+Neon maintains 100% compatibility with any application that uses the official Postgres release.
+That makes Neon a good option to consider as the database backend for FerretDB, while leveraging Neon's native features to scale and manage your database infrastructure easily.
+
+In this blog post, you'll learn about FerretDB and and how you can add MongoDB compatibility to Neon.
+
+## Advantages of using FerretDB
+
+Let's take a look at some of these benefits of using FerretDB:
+
+### MongoDB compatibility
+
+The best advantage to using FerretDB is the access you get to the syntax, tools, querying language, and commands available in MongoDB, particularly for many common use cases.
+MongoDB is known for its simple and intuitive NoSQL query language which is widely loved by many developers today.
+By using FerretDB, you can enable Postgres-compatible databases like Neon to run MongoDB workloads.
+
+Read more: [MongoDB Compatibility - What's Really Important?](https://blog.ferretdb.io/mongodb-compatibility-whats-really-important/)
+
+### Open source
+
+As an open source document database, you won't be at a risk of vendor lock-in.
+Since MongoDB's license change to Server Side Public License (SSPL), there's [been a lot of disappointment and confusion with regards](https://blog.ferretdb.io/open-source-is-in-danger/) to how it affects many users and what it would mean for their applications.
+According to the Open Source Initiative – the steward of open source and the set of rules that define open source software – SSPL is not considered open source.
+
+In that regard, FerretDB, licensed under Apache 2.0, is a good option for users looking for a MongoDB alternative to return MongoDB workloads back to open source.
+
+### Multiple backend options
+
+At the moment, FerretDB supports Postgres and SQLite backends, with many ongoing works to support other backends.
+Many databases built on Postgres can actually serve as a backend for FerretDB, including Neon.
+That means you can take advantage of all the features available in that backend to scale and manage your database infrastructure without any fear of vendor lock-in.
+
+## Prerequisites
+
+- A Neon project and Postgres URI
+- FerretDB package
```

Comment:
What is a FerretDB package? Maybe "Install FerretDB" or a link to the release pages would make this clearer.